ml-agents/mlagents/trainers/ppo/trainer.py

```diff
@@ -181,6 +160,12 @@ def add_experiences(self, curr_all_info: AllBrainInfo, next_all_info: AllBrainIn
         :param next_all_info: Dictionary of all current brains and corresponding BrainInfo.
         :param take_action_outputs: The outputs of the take action method.
```

Comment:
Fix comment to make it explicit that we are referencing a method of a different class now that `take_action` is not part of `Trainer` type.